src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -215,6 +215,11 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                 ->prototype('scalar')->end()
             ->end()
             ->booleanNode('security')->defaultTrue()->end()
+            ->arrayNode('user_checkers')
+                ->defaultValue(array('security.user_checker'))
+                ->info('A list of user checkers reserved for this firewall.')
```

Comment:
A list of user checker service ids to use when authenticating users in this firewall.